### ISSUE TITLE
Clipping on unit cell box

### ISSUE THREAD

**Issue by mkhorton** [3dmol/3Dmol.js#304]
When drawing the unit cell box, the box outline is often clipped from certain camera angles. Image attached.

<img width="497" alt="clipping" src="https://cloud.githubusercontent.com/assets/2976580/26594725/9b9cd57e-451d-11e7-8393-ec9126a9756c.png">

**Comment by dkoes** [comment]
Did you call zoomTo _after_ creating the unit cell box?

**Comment by mkhorton** [comment]
Yes, is that correct? I was using some code from the tests:

```javascript
var model = $scope.MAIN_VIEWER.addModel(data, format);
$scope.MAIN_VIEWER.addUnitCell(model);
model.setStyle({}, {sphere:{scale: 0.2, colorscheme:"Jmol"}, stick:{radius:0.1, colorscheme:"Jmol"}});	  
$scope.MAIN_VIEWER.zoomTo();
$scope.MAIN_VIEWER.render();
```

**Comment by dkoes** [comment]
Fixed 62cd9a9
addLine wasn't updating boundarySphere information and so the box was being ignored in zoomTo.
